bitwarden_license/src/Scim/Models/ScimUserRequestModel.cs

```diff
@@ -25,6 +26,14 @@ public OrganizationUserInvite ToOrganizationUserInvite(ScimProviderType scimProv
         };
     }
 
+    public OrganizationUserSingleEmailInvite ToInvite(ScimProviderType scimProvider, bool hasSecretsManager) =>
+        OrganizationUserSingleEmailInvite.Create(
+            EmailForInvite(scimProvider),
+            [],
+            OrganizationUserType.User,
+            new Permissions(),
+            hasSecretsManager);
```

Comment:
I prefer to name the parameters, its easier to read
```suggestion
        OrganizationUserSingleEmailInvite.Create(
            email: EmailForInvite(scimProvider),
            accessibleCollections: [],
            type: OrganizationUserType.User,
            permissions: new Permissions(),
            accessSecretsManager: hasSecretsManager);
```